cordova-lib/src/cordova/platform.js

```diff
@@ -68,6 +68,13 @@ function addHelper(cmd, hooksRunner, projectRoot, targets, opts) {
         }
     }
 
+    if (opts.usegit) {
+        msg = '\r\nWARNING: The --usegit flag has been deprecated ! \r\n' +
+              'instead, please use: `cordova platform add git-url#custom-branch`. \r\n' +
+              'e.g: cordova platform add https://github.com/apache/cordova-android.git#2.4.0 \r\n';
+        events.emit('log', msg);
```

Comment:
Some nits:
* Use `\n`, not `\r\n`.
* Use `\'` rather than backtick (`` ` ``).
* Use `events.emit('warning'...)`
* No space before `!` (or even just a period - no need to get too excited :smile:).
* Capital `I` for `Instead`